core/controllers/questions_list.py

```diff
@@ -63,10 +67,20 @@ def get(self, comma_separated_skill_ids):
 
         (
             question_summaries, merged_question_skill_links,
-            next_start_cursor) = (
-                question_services.get_displayable_question_skill_link_details(
-                    constants.NUM_QUESTIONS_PER_PAGE, skill_ids, start_cursor)
-            )
+            next_offset
+        ) = (
+            question_services.get_displayable_question_skill_link_details(
+                constants.NUM_QUESTIONS_PER_PAGE + 1, skill_ids, offset)
+        )
+        next_offset -= min(len(skill_ids), constants.MAX_SKILLS_PER_QUESTION)
+        # Set next_offset to None if there are no more questions so that
```

Comment:
See a previous comment I made, consider adjusting this so that the return value is a boolean instead.